Playground handbook (#2094)
microsoft/TypeScript-Website: packages/playground-handbook/Overview.md

## Welcome to the Playground Handbook

> The TypeScript playground is an online environment where people can write and share TypeScript-ish code. We say "ish" because you can work with `.ts`, `.tsx`, `.js`, `.jsx` and `.d.ts` files inside the playground. The goal for the Playground is to be a safe environment which requires no setup, is a single text document, can be trivially shared with others, where URLs still works years down the line and it makes a teaching tool where you can safely experiment.

Reviewer: ```suggestion
The TypeScript playground is an online environment where people can write and share TypeScript-ish code. We say "ish" because you can work with `.ts`, `.tsx`, `.js`, `.jsx` and `.d.ts` files inside the playground. The goal for the Playground is to be a safe environment which requires no setup, is a single text document, can be trivially shared with others, where URLs still works years down the line. In summary, a teaching tool where you can safely experiment.
```

> From [day one in 2012](https://web.archive.org/web/20121031123957/http://www.typescriptlang.org/Playground/), the TypeScript website has featured a playground as a way to highlight the difference between the TypeScript code you write and the JavaScript which is emitted. Today, the Playground has a massive set of features because the needs of developers using TypeScript has grown in scope - there's over a hundred [`tsconfig.json`](https://www.typescriptlang.org/tsconfig) flags which effect everything from the strictness of the type-checker to some very exacting JavaScript needs. Developers need a safe way to be able to reproduce a particular TypeScript environment which can be shared with others.

Reviewer: ```suggestion
From [day one in 2012](https://web.archive.org/web/20121031123957/http://www.typescriptlang.org/Playground/), the TypeScript website has featured a playground as a way to highlight the difference between the TypeScript code you write and the JavaScript which is emitted. Today, the Playground has a massive set of features because the needs of developers using TypeScript has grown in scope - there's over a hundred [`tsconfig.json`](https://www.typescriptlang.org/tsconfig) flags which affect everything from the strictness of the type-checker to some very exacting JavaScript needs. Developers need a safe way to be able to reproduce a particular TypeScript environment which can be shared with others.
```

I would actually drop everything after 'tsconfig.json flags'
